components/ui/card.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react'
+import { cn } from '@/lib/utils'
+import { CardI } from '@/lib/interface/controls/card'
+import Image from 'next/image'
+
+const Card = React.forwardRef<HTMLDivElement, CardI>(
+  (
+    {
+      className,
+      title,
+      children,
+      variant = 'default',
+      primaryColor = 'default',
+      imgUrl,
+      members,
+      subject,
+      desc,
+      Icon,
+      ...props
+    },
+    ref
+  ) => (
+    <>
+      {variant === 'default' && (
+        <div
+          ref={ref}
+          className={cn(
+            'rounded-lg border border-slate-200 bg-white  min-w-[280px] max-w-[300px] h-[160px] text-slate-950 shadow-sm dark:border-slate-800 dark:bg-slate-950 dark:text-slate-50',
+            className
+          )}
+          {...props}
+        >
+          <CardHeader
+            className={`${primaryColor == 'default' ? 'bg-[#3A86FF]' : 'bg-[#EECB51]'} font-medium rounded-t-lg flex-row   flex items-center gap-2`}
+          >
+            <div
+              className={`${primaryColor == 'default' ? 'bg-[#66A1FF]' : 'bg-[#FCEBAF]'}
+        text-white px-2 py-2 rounded-tl-lg`}
+            >
+              <Icon />
+            </div>
+            <p>
+              {title.length > 0 && (
+                <CardDescription className="text-white">
+                  {title}
+                </CardDescription>
+              )}
+            </p>
+          </CardHeader>
+          <CardContent>{children}</CardContent>
+        </div>
+      )}
+      {variant === 'image' && (
+        <div
+          className={cn(
+            'rounded-lg border border-slate-200 bg-white  min-w-[300px] pb-[22px] max-w-[320px] min-h-[260px] text-slate-950 shadow-sm dark:border-slate-800 dark:bg-slate-950 dark:text-slate-50',
```

Comment:
As before, this line is too long. Break it up with a max length of 80 chars, and spread it out over multiple lines. It would also be nice to use `_uniq(...)` on the final class list to ensure the user does not supply classes already defined here in the `className` prop. That would look like `className={_uniq(cn(...))}`. Import it from `lodash` with `import _uniq from 'lodash/uniq'`.